assets/src/components/Dashboard.tsx

```diff
@@ -51,14 +51,28 @@ const Dashboard = (props: RouteComponentProps) => {
 
   const logout = () => auth.logout().then(() => props.history.push('/login'));
 
+  const [notificationMessage, setNotificationMessage] = useState('Papercups');
+
+  const INTERVAL = 2000; //Set flashing interval
+
+  const _toggleNotificationMessage = () => {
```

Comment:
nitpick -- can we remove the `_` at the beginning of this function name? (it seems unnecessary)

Reply:
I will remove this on next commit